deploy-board/deploy_board/static/js/components/capacitycomponents.js

```diff
@@ -345,23 +344,23 @@ Vue.component("static-capacity-config", {
                                 xhr.setRequestHeader("X-CSRFToken", csrftoken);
                             },
                         });
-                        //console.log(result);
                         return result.length;
                     } catch (error) {
                         console.error(error);
                         return 0;
                     }
                 };
-                doAjax().then( (data) => terminatingHostCount = data );
-                //console.log(terminatingHostCount);
-
-                if (-sizeIncrease > this.terminationLimit) {
-                    this.showTerminationError = true;
-                    this.terminationError = getTerminationLimitAlertMessage(false);
-                } else if (-sizeIncrease > this.terminationLimit - terminatingHostCount) {
-                    this.showTerminationError = true;
-                    this.terminationError = getTerminationLimitAlertMessage(true);
+                doAjax().then( (data) => {
```

Comment:
Since you are improving this. I'd suggest a more complete solution.
```suggestion
fetch(url, {
      headers: {
        "Content-Type": "application/json",
      },
    })
.then((terminatingHosts) =>
 {
   ...
 }
).catch(function(error){
  console.log(error);
});;
```